app/Service/StudentService.php

```diff
@@ -16,7 +16,7 @@ public function createStudents(Reader $csvReader)
         $studentsToInsert = [];
         try
         {
-            $students = $csvReader->getRecords();
+            $students = iterator_to_array($csvReader->getRecords());
```

Comment:
we didn't take a look to this modification. It worked, but I don't know if you wanted that way.

Reply:
Please remove this and revert back to the original.